org.opent2t.sample.hub.superpopular/com.wink.hub/test/mockWinkHub.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+var _ = require('lodash');
```

Comment:
I wanted the deep equals functionality (line 16 below).  lodash is used by Ava for this purpose already, so it doesn't add any additional packaged to node_modules.

---
In reply to: [106019229](https://github.com/openT2T/translators/pull/209#discussion_r106019229) [](ancestors = 106019229)